server/utils/firebase_algolia_server_code/functions/test/algoliaMocks.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Add an altered field to the json data passed into the mock functions. 
+ * Use this field to validate that the data returned by the cloud functions
+ * Corresponds to the data returned by the algoliaOperation input. 
+ * @param {Object} someJson an arbitrary json input
+ * @return {Object}
+ */
+function addField(someJson) {
```

Comment:
You should let users specify key/value to change.  Otherwise they may call this twice expecting it to change, and it doesn't change.

Or if all this is doing is marking the JSON as modified, change the method name to make that more clear.